meson.build

```diff
@@ -40,12 +40,14 @@ project('picolibc', 'c',
 	  'b_staticpic=false',
 	],
 	license : 'BSD',
-	meson_version : '>= 0.50',
+	meson_version : '>= 0.53.2',
 	version: '1.7.3'
        )
 
 targets = []
 
+cc = meson.get_compiler('c')
+
```

Comment:
Yeah, I'd been meaning to do this for a very long time. Thanks for taking the time!